src/electos/ballotmaker/make_ballots.py

```diff
@@ -0,0 +1,18 @@
+"""Generate ballots from EDF data
+    Requires: 
+        EDF file (JSON format) edf_file: Path, 
+    Optional:
+        Output directory for generated PDF files
+        Settings file for ballot generation
```

Comment:
Better to document the detailed arguments on the function, not the module.

Reply:
Agreed and done.